cmd/skopeo/proxy.go

```diff
@@ -235,7 +235,8 @@ func isDockerManifestUnknownError(err error) bool {
 // TODO drive this into containers/image properly
 func isNotFoundImageError(err error) bool {
 	return isDockerManifestUnknownError(err) ||
-		errors.Is(err, ocilayout.ImageNotFoundError{})
+		errors.Is(err, ocilayout.ImageNotFoundError{}) ||
+		errors.Is(err, os.ErrNotExist)
```

Comment:
OK self-review...perhaps the real fix is in c/image to map enoent there to `ocilayout.ImageNotFoundError` ?